docs/config/index.md

```diff
@@ -462,6 +462,13 @@ export default defineConfig(({ command, mode }) => {
   `envPrefix` should not be set as `''`, which will expose all your env variables and cause unexpected leaking of of sensitive information. Vite will throw error when detecting `''`.
   :::
 
+### spa
+
+- **Type:** `boolean`
+- **Default:** `true`
+
+  Whether your application is a Single Page Application (SPA). Set to `false` for other kinds of apps like MPAs. Learn more in Vite's [SSR guide](/guide/ssr#vite-cli).
```

Comment:
Should this link have an `.html` ending? The link in the next `.md` file does, so I figure one of the two is probably wrong, but I'm not sure which one

Reply:
I don't know. @patak-dev ?

Reply:
(I've seen both in the docs, so I was assuming it doesn't matter. But I guess patak will know more about this.)

Reply:
It isn't needed 👍🏼